bundles/AdminBundle/src/Security/Authenticator/PreAuthenticatedAdminSessionAuthenticator.php

```diff
@@ -86,13 +86,8 @@ public function supports(Request $request): ?bool
 
     public function authenticate(Request $request): Passport
     {
-        $method = 'loadUserByIdentifier';
-        if (!method_exists($this->userProvider, 'loadUserByIdentifier')) {
-            $method = 'loadUserByUsername';
-        }
-
         return new SelfValidatingPassport(
-            new UserBadge($request->attributes->get('_pre_authenticated_username'), [$this->userProvider, $method]),
+            new UserBadge($request->attributes->get('_pre_authenticated_username'), [$this->userProvider, 'loadUserByIdentifier']),
```

Comment:
Since we're on PHP ^8.1 we can use the [first-class callable syntax](https://www.php.net/manual/en/functions.first_class_callable_syntax.php) here:

```suggestion
            new UserBadge($request->attributes->get('_pre_authenticated_username'), $this->userProvider->loadUserByIdentifier(...)),
```